Mathlib/Tactic/Abel.lean

```diff
@@ -497,37 +498,14 @@ partial def abelNFCore
   withConfig ({ · with zetaDelta := cfg.zetaDelta }) <| go true e
 
 open Parser.Tactic
-/-- Use `abel_nf` to rewrite the main goal. -/
-def abelNFTarget (s : IO.Ref AtomM.State) (cfg : AbelNF.Config) : TacticM Unit := withMainContext do
-  let goal ← getMainGoal
-  let tgt ← withReducible goal.getType'
```

Comment:
Flagging for a final look: the new code does `instantiateMVars` (instead of `withReducible`).